flutter_local_notifications/example/lib/main.dart

```diff
@@ -316,6 +316,14 @@ class _HomePageState extends State<HomePage> {
                         await _scheduleDailyTenAMNotification();
                       },
                     ),
+                    PaddedRaisedButton(
+                      buttonText:
+                          'Schedule daily 10:00:00 am notification in your '
+                          'local time zone given last year',
```

Comment:
there's a grammar issue with ending this sentence with `given last year`.  Unless you have another example that doesn't have a grammar issue, `...using last year's date` that  I gave as an example before should work

Reply:
fixed